src/Hit/Cli.hs

```diff
@@ -89,7 +89,11 @@ freshP :: Parser HitCommand
 freshP = Fresh <$> maybeBranchP
 
 newP :: Parser HitCommand
-newP = New <$> issueNumP
+newP = New <$> issueNumP <*> assignOwnerP
+
+assignOwnerP :: Parser Bool
+assignOwnerP = switch
```

Comment:
Is it `False by default? Can we add documentation mentioning that here?

Reply:
I'm not sure why GitHub is no showing the diff here, but documentation has been added.